src/main/java/Duke.java

```diff
@@ -1,10 +1,174 @@
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    public static final int SKIP_DEADLINE_NUMBER = 9;
+    public static final int SKIP_EVENT_NUMBER = 6;
+    public static final int SKIP_DONE_NUMBER = 5;
+    public static final int SKIP_SEPARATION_NUMBER = 4;
+
+    public static ArrayList<Task> list = new ArrayList<>();
```

Comment:
Consider using a more descriptive name for your ArrayList to clearly show what it is a list of